src/components/select/multi-select/multi-select.mdx

```diff
@@ -143,6 +143,10 @@ be customised if desired using the `virtualScrollOverscan` prop. Higher values w
 
 <Canvas of={MultiSelectStories.Virtualised} />
 
+## Testing with RTL
```

Comment:
**suggestion(non-blocking)**: similarly, what do you think about including the full name too, since its the first time the acronym is used?

```suggestion
## Testing with React Testing Library (RTL)
```